frontend/scripts/migrate-order-index.ts

```
@@ -0,0 +1,150 @@
+  data: T;
+  meta: {
+    pagination: {
+      page: number;
+      pageSize: number;
+      pageCount: number;
+      total: number;
+    };
+  };
+}
+
+async function fetchAPI(path: string, options: any = {}) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)